tests.toml

```diff
@@ -0,0 +1,34 @@
+#:schema https://raw.githubusercontent.com/YunoHost/apps/master/schemas/tests.v1.schema.json
+
+test_format = 1.0
+
+[default]
+
+    # ------------
+    # Tests to run
+    # ------------
+
+    exclude = ["install.subdir", "install.multi"]
```

Comment:
These are not run already because the app doesnt have a `path` install question, and multi_instance is set to false ...
```suggestion
```